README.md

```diff
@@ -0,0 +1,17 @@
+# Spring Boot tutorial
```

Comment:
update readme. include more information regarding project. also include various steps to run check style, build etc..